npmVowelCounter/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (str) => {
```

Comment:
When the function is called without any input (ie: vowelCount() ) it returns a type error. You'll need to check that an input parameter exists.